docs/usage/locators.md

```diff
@@ -166,6 +166,30 @@ element(by.control({
   }
 }));
 ```
+### Common usecases
+
+#### Working with responsive toolbars
+A responsive toolbar may have overflowing content, depending on the screen size.
+This content will be moved to a popover which is open by pressing a toggle button in the toolbar.
+A toggle button will be shown only when there is overflowing content. This is a problem for tests because they should only try to press the button when it's visible and interactable. One way to solve this is to always start the application under test with with a fixed screen size. Another way is to first look for any toggle button - with no restriction on visibility, and then press on it only if it exists:
+```javascript
+var button = element(by.control({
```

Comment:
have you tested this (why not add а test) ? IMHO this won't work because we have an implicit visible: true selector and so button will be null in case the toggle button is not displayed.

Reply:
tested